docs/digger/admin/known-issues.asciidoc

```diff
@@ -1,2 +1,2 @@
 == Known issues
```

Comment:
this pr will create conflict with https://github.com/aerogear/aerogear.org/pull/683/files I will try to handle this

Reply:
@pmacko1 it would be better to merge this and rebase your pr on top of it, resolving the conflicts in this rebase.